[2024-05-28T02:47:40.401+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_load_stage_data manual__2024-05-27T21:49:08.726117+00:00 [queued]>
[2024-05-28T02:47:40.420+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_load_stage_data manual__2024-05-27T21:49:08.726117+00:00 [queued]>
[2024-05-28T02:47:40.420+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T02:47:40.420+0000] {taskinstance.py:1284} INFO - Starting attempt 20 of 20
[2024-05-28T02:47:40.420+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T02:47:40.440+0000] {taskinstance.py:1304} INFO - Executing <Task(DockerOperator): ti_load_stage_data> on 2024-05-27 21:49:08.726117+00:00
[2024-05-28T02:47:40.447+0000] {standard_task_runner.py:55} INFO - Started process 1629 to run task
[2024-05-28T02:47:40.450+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'franko_lookup_ingest', 'ti_load_stage_data', 'manual__2024-05-27T21:49:08.726117+00:00', '--job-id', '233', '--raw', '--subdir', 'DAGS_FOLDER/challenge_dag.py', '--cfg-path', '/tmp/tmp09bfr0on']
[2024-05-28T02:47:40.451+0000] {standard_task_runner.py:83} INFO - Job 233: Subtask ti_load_stage_data
[2024-05-28T02:47:40.554+0000] {task_command.py:389} INFO - Running <TaskInstance: franko_lookup_ingest.ti_load_stage_data manual__2024-05-27T21:49:08.726117+00:00 [running]> on host a64b6410e10e
[2024-05-28T02:47:40.665+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=franko_lookup_ingest
AIRFLOW_CTX_TASK_ID=ti_load_stage_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-27T21:49:08.726117+00:00
AIRFLOW_CTX_TRY_NUMBER=20
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-27T21:49:08.726117+00:00
[2024-05-28T02:47:40.713+0000] {docker.py:284} INFO - Starting docker container from image my_dbt
[2024-05-28T02:47:43.406+0000] {docker.py:354} INFO - [0m02:47:43  Running with dbt=1.5.11
[2024-05-28T02:47:43.563+0000] {docker.py:354} INFO - [0m02:47:43  Registered adapter: postgres=1.5.11
[2024-05-28T02:47:43.728+0000] {docker.py:354} INFO - [0m02:47:43  Found 6 models, 4 tests, 0 snapshots, 0 analyses, 422 macros, 0 operations, 1 seed file, 1 source, 0 exposures, 0 metrics, 0 groups
[2024-05-28T02:47:43.730+0000] {docker.py:354} INFO - [0m02:47:43
[2024-05-28T02:47:43.818+0000] {docker.py:354} INFO - [0m02:47:43  Concurrency: 4 threads (target='dev')
[2024-05-28T02:47:43.818+0000] {docker.py:354} INFO - [0m02:47:43
[2024-05-28T02:47:43.822+0000] {docker.py:354} INFO - [0m02:47:43  1 of 2 START sql table model dev.silver_covid_geo .............................. [RUN]
[2024-05-28T02:47:44.321+0000] {docker.py:354} INFO - [0m02:47:44  1 of 2 OK created sql table model dev.silver_covid_geo ......................... [[32mSELECT 283[0m in 0.50s]
[2024-05-28T02:47:44.324+0000] {docker.py:354} INFO - [0m02:47:44  2 of 2 START sql view model dev.silver_unpivot_covid_geo ....................... [RUN]
[2024-05-28T02:47:46.110+0000] {docker.py:354} INFO - [0m02:47:46  2 of 2 OK created sql view model dev.silver_unpivot_covid_geo .................. [[32mCREATE VIEW[0m in 1.78s]
[2024-05-28T02:47:46.162+0000] {docker.py:354} INFO - [0m02:47:46
[2024-05-28T02:47:46.162+0000] {docker.py:354} INFO - [0m02:47:46  Finished running 1 table model, 1 view model in 0 hours 0 minutes and 2.43 seconds (2.43s).
[2024-05-28T02:47:46.199+0000] {docker.py:354} INFO - [0m02:47:46
[2024-05-28T02:47:46.199+0000] {docker.py:354} INFO - [0m02:47:46  [32mCompleted successfully[0m
[2024-05-28T02:47:46.200+0000] {docker.py:354} INFO - [0m02:47:46
[2024-05-28T02:47:46.200+0000] {docker.py:354} INFO - [0m02:47:46  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-05-28T02:47:47.005+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=franko_lookup_ingest, task_id=ti_load_stage_data, execution_date=20240527T214908, start_date=20240528T024740, end_date=20240528T024747
[2024-05-28T02:47:47.047+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-05-28T02:47:47.081+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
